packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -556,8 +554,7 @@ export const ListFeatures: Record<
     BuildInEditFeature<PluginKeyboardEvent>
 > = {
     autoBullet: AutoBullet,
-    indentWhenTab: IndentWhenTab,
-    outdentWhenShiftTab: OutdentWhenShiftTab,
```

Comment:
This is already a public interface, remove item from it will cause a breaking change

Reply:
Ok, I'll add a new feature